### PR TITLE
5068 discord bot send message with file

### PR DESCRIPTION
**Pull request by alysonturing**

### CODE REVIEW

File: components/discord_bot/actions/send-message-with-file/send-message-with-file.mjs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      type: "boolean",
+      optional: true,
+      default: true,
+      label: "Include link to workflow",
+      description: "Defaults to `true`, includes a link to this workflow at the end of your Discord message.",
+    },
+  },
+  methods: {
+    ...common.methods,
+    getUserInputProps(omit = [
```
Comment (luancazarine) [inline components/discord_bot/actions/send-message-with-file/send-message-with-file.mjs:85]: It looks like you're not using this method, so you can remove it.

**Review by luancazarine — CHANGES_REQUESTED** [review]
Hello @alysonfreitas. thanks for your contribution! I've just left a minor adjustment request.

File: components/discord_bot/actions/send-message-with-file/send-message-with-file.mjs (hunk 1 above)
Comment (alysonturing) [reply components/discord_bot/actions/send-message-with-file/send-message-with-file.mjs]: done!

**Review by luancazarine — APPROVED** [review]
LGTM!

**Pull request merged**